### DISCUSSION TITLE
How add server side graphql subscriptions

### DISCUSSION THREAD

**Discussion by theEngMansour** [vercel/next.js#42211] [Help]
### Describe the feature you'd like to request

How add server side graphql subscriptions to Next.js,I am currently building app in Next JS. I use graphql with Apollo Server micro on backend. Now I want real time updates, but I found out there is no support for Subscription in apollo-server-micro

### Describe the solution you'd like

/

### Describe alternatives you've considered

/

**Comment by satoshinakamoto501** [comment]
get the real Value of your app thought by computing it's digital value of each word of your app.. then it'll tell you the solution there. use your sense of π
